pkg/contexts/ocm/repositories/comparch/uniform.go

```diff
@@ -11,13 +11,9 @@ import (
 	"github.com/open-component-model/ocm/pkg/contexts/ocm/cpi"
 )
 
-const AltType = "ca"
-
```

Comment:
Because this was al alternate for the ref parsing, there was no registered ShortType.